### PR TITLE
refactor(rust): update how we handle user confirmation on `reset` command

### PR DESCRIPTION
**Pull request by adrianbenavides**

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/reset.rs [hunk 1]
```diff
@@ -11,33 +13,32 @@ pub struct ResetCommand {
...
+            std::process::exit(e.code());
         }
     }
 }
 
-fn run_impl(opts: CommandGlobalOpts) -> crate::Result<()> {
+fn run_impl(opts: CommandGlobalOpts, cmd: ResetCommand) -> crate::Result<()> {
+    match opts
+        .terminal
+        .confirm("This will delete the local CLI state. Are you sure?")?
```
Comment (mrinalwadhwa) [inline implementations/rust/ockam/ockam_command/src/reset.rs]: We need a better message here. CLI state has no meaning to end users.

Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/reset.rs]: I changed it to "local Ockam configuration". The point I want to make is that the command only affects local stuff.

Comment (etorreborre) [reply implementations/rust/ockam/ockam_command/src/reset.rs]: @mrinalwadhwa are you ok with that wording? Can we merge?

Comment (mrinalwadhwa) [reply implementations/rust/ockam/ockam_command/src/reset.rs]: That change is good but there is another reference to CLI state in this PR

Comment (etorreborre) [reply implementations/rust/ockam/ockam_command/src/reset.rs]: Right, in the last commit I changed the wording everywhere CLI state was used

Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/reset.rs]: Thanks @etorreborre ! I didn't see the other reference and forgot to follow up on this

**Review by metaclips — APPROVED** [review]

**Pull request merged**